src/common/log/log.ts

```diff
@@ -50,7 +50,7 @@ export module Log {
      * them to them.
      */
     export function logInternalMessage(logLevel: LogLevel, message: string) {
-        if (LogHelper.logLevel >= logLevel) {
+        if (LogHelper.internalLogLevel >= logLevel) {
```

Comment:
I considered that option too, but given that we are using the internal logging as a complete different channel (because we can't print to the Chrome Developer Tools console), I think it makes more sense to have 2 different logging channels and levels. I can't see any scenario where a normal user would benefit from seeing this logs, unless he is actually developing/debugging the extension itself. If we go that route I think we'll end up with 10 different log levels with weird names, that will only confuse the users.